apps/vscode/values.yaml

```diff
@@ -13,7 +13,7 @@ apps:
   volumek8s:
 
 appconfig:
-  image: codercom/code-server:latest
+  image: ghcr.io/scilifelabdatacentre/serve-vscode:250312-1210
```

Comment:
I think it's okay if we set it to latest. It's just us creating and updating the image. But I'm not insisting